modules/virt-cloning-vm-snapshot.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-backup-restore-snapshots.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="virt-cloning-vm-snapshot_{context}"]
+= Cloning a snapshot by using the web console
+
+You can clone an existing virtual machine (VM) snapshot.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines* in the web console.
+. Select a VM to open the *VirtualMachine details* page.
+. Click the *Snapshots* tab.
+. Click the options menu {kebab} for the snapshot you want to clone and select *Clone*.
+. On the *Clone VirtualMachineSnapshot* page, enter the snapshot name.
```

Comment:
Here it seems like we are creating a new snapshot based on the original one. Instead, we are creating a VM.  In the step here we have `enter the snapshot name.` but instead we are entering the new VM name

Reply:
This procedure has changed and is now **Cloning a VM by using the web console**, with updated steps